docs/content/faq.md

```diff
@@ -250,23 +249,16 @@ See [#1247][github-issue-1247] for an example.
 
 ### Common Errors
 
+Due to the way we set `mydestination`, you might encounter the following issue:
+
 ```log
-warning: connect to Milter service inet:localhost:8893: Connection refused
-# DMARC not running
-# => /etc/init.d/opendmarc restart
-
-warning: connect to Milter service inet:localhost:8891: Connection refused
-# DKIM not running
-# => /etc/init.d/opendkim restart
-
-mail amavis[1459]: (01459-01) (!)connect to /var/run/clamav/clamd.ctl failed, attempt #1: Can't connect to a UNIX socket /var/run/clamav/clamd.ctl: No such file or directory
-mail amavis[1459]: (01459-01) (!)ClamAV-clamd: All attempts (1) failed connecting to /var/run/clamav/clamd.ctl, retrying (2)
-mail amavis[1459]: (01459-01) (!)ClamAV-clamscan av-scanner FAILED: /usr/bin/clamscan KILLED, signal 9 (0009) at (eval 100) line 905.
-mail amavis[1459]: (01459-01) (!!)AV: ALL VIRUS SCANNERS FAILED
-# Clamav is not running (not started or because you don't have enough memory)
-# => check requirements and/or start Clamav
+warning: do not list domain email.example.com in BOTH mydestination and virtual_mailbox_domains
+...
+NOQUEUE: reject: RCPT from HOST[IP]: 550 5.1.1 <RECIPIENT>: Recipient address rejected: User unknown in local recipient table; ...
 ```
 
+You may solve this problem by running `sed -i 's|mydestination\s=\s\$myhostname,\slocalhost\.\$mydomain,\slocalhost|mydestination=localhost.$mydomain,localhost|g' /etc/postfix/main.cf` in [`user-patches.sh`][docs-userpatches].
```

Comment:
Wouldn't it be simpler to use `postfix-main.cf` override instead?

````suggestion
Opt-out of mail being directed to services by excluding `$myhostname` as a destination with a [`postfix-main.cf`][docs-override-postfix] override config:

```cf
mydestination = localhost.$mydomain, localhost
```

!!! tip

    You may want to configure a `postmaster` alias via `setup alias add` to receive system notifications.

!!! warning

    Internal mail destined for `root`, `amavis` or other accounts will now no longer be received without an alias or account created for them.

[docs-override-postfix]: ./config/advanced/override-defaults/postfix.md
````